docs/index.html

```diff
@@ -792,6 +794,26 @@ <h3 class="i18n" id="trx_color">Custom Colors</h3>
     </p>
     <hr>
 
+    <h2 class="i18n">Finding Transactions</h2>
+        <div class="row">
+        </div>
+            <p class="i18n" id="trx_search">
+                The MMEX desktop application provides several utilities to help users find specific transactions. The most powerful of these tools is the Transaction Reports
+                feature described in more detail <a class="i18n" href="#section21.1">here</a>. In addition, there are several other utilities which
+                can be used to find transactions.
+            </p>
+        
+        <hr>
+    
+        <h3 class="i18n" id="trx_search-1">Searching from the Account View</h3>
```

Comment:
class="i18n" id="trx_search-1" are deprecated
Please use clear <h2> or <h3> tags for header

Reply:
@vomikan Thank you for the guidance on appropriate HTML styling! I will add general header cleanup to my to-do list as well to help bring some uniformity.